script/install.sh

```diff
@@ -9,9 +9,11 @@ _() {
 
 # Returns false if the script is invoked from a Windows command prompt.
 posix_terminal() {
-    # If this script is run on Windows cmd by passing it as argument to bash.exe,
-    # the shell level will be 1. If it is run from a POSIX terminal, it will be > 1.
-    if [ "$SHLVL" = 1 ]; then
+    # If run from a POSIX terminal (including under MSYS2 or Cygwin) the TERM
+    # variable is defined to something like "xterm". If run from a Windows
+    # command prompt through bash.exe from an MSYS installation, TERM keeps
+    # its default value, which is "cygwin".
+    if [[ $TERM == "cygwin" ]]; then
         false
```

Comment:
there are no booleans in bash!

```suggestion
        return 0
```

Reply:
But you got it backwards. `return 0` == `true` in bash, unlike D. https://stackoverflow.com/questions/23320150/bash-boolean-testing/23324368#23324368. I believe I had it this way in my first iteration and you commented rightfully that it was confusing. Good or bad style, at least this cannot be misunderstood.